src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -25,6 +35,70 @@ static void EnsureLibSsl10Initialized()
 }
 #endif
 
+static int32_t g_config_specified_ciphersuites = 0;
+
+static void DetectCiphersuiteConfiguration()
+{
+    // This routine will always produce g_config_specified_ciphersuites = 0 on OpenSSL 1.0.x,
+    // so if we're building direct for 1.0.x (the only time NEED_OPENSSL_1_1 is undefined) then
+    // just omit all the code here.
+    //
+    // The method uses OpenSSL 1.0.x API, except for the fallback function SSL_CTX_config, to
+    // make the portable version easier.
+#ifdef NEED_OPENSSL_1_1
+
+    // Check to see if there's a registered default CipherString. If not, we will use our own.
+    SSL_CTX* ctx = SSL_CTX_new(TLS_method());
```

Comment:
Can this and other `SSL_*_new` calls in this method ever return _nullptr_?

Reply:
Only when malloc of a little number does.  Added some asserts so the shutdown is cleaner than SIGSEGV.